CONCURRENCY.md

```diff
@@ -207,4 +219,4 @@ that are potentially parts of shared cyclic data should be zeroed out once no lo
  If atomic reference value is attempted to be set to non-frozen value runtime exception is thrown.
 
  Freezable atomic reference is similar to the regular atomic reference, but until frozen behaves like regular box
```

Comment:
Freezable atomic reference is similar to the regular atomic reference, but until frozen behaves like a regular box

what is a regular box?

Reply:
I haven't updated content in this article. Just fixed formatting. No proofread is required for that article for now. Thx!